src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -19,7 +20,7 @@ public static string FrameworkDescription
         {
             get
             {
-                return FrameworkName;
+                return string.Format("{0} {1}", FrameworkName, typeof(object).GetTypeInfo().Assembly.GetName().Version.ToString());
```

Comment:
nit: I think you don't technically need the `ToString()` call because `string.Format` will do it for you.